### ISSUE TITLE
Find root cause of catkin build failing

### ISSUE THREAD

**Issue by PerrineAguiar** [Intermodalics/tango_ros#210]
We currently use `catkin build --no-jobserver` as a workaround.

**Comment by mcopejans** [comment]
The following error also occurs when you do build with `--no-jobserver`.

```
make[2]: *** No rule to make target make[2]: *** No rule to make target `/home/mcopejans/tangoscan_ws/devel/.private/tango_ros_messages/include/tango_ros_messages/SaveMapRequest.h', needed by `/home/mcopejans/tangoscan_ws/src/tango_ros/tango_ros_common/tango_ros_messages/include/tango_ros_messages/SaveMapRequest.h'.  Stop.
```

**Comment by PerrineAguiar** [comment]
This was fixed in #191.
